natural_language_processing/extractive_question_answering/bert_large/run_huggingface.py

```diff
@@ -15,7 +15,7 @@
 def parse_args():
     parser = argparse.ArgumentParser(description="Run model from Huggingface's transformers repo for extractive question answering task.")
     parser.add_argument("-m", "--model_name",
-                        type=str, choices=["bert-large-uncased-whole-word-masking-finetuned-squad"], required=True,
+                        type=str, default="bert-large-uncased-whole-word-masking-finetuned-squad", required=True,
```

Comment:
This is a directory dedicated to bert_large, could you introduce other architectures as separate scripts? Bert Bases can go here btw: https://github.com/AmpereComputingAI/ampere_model_library/blob/18baa174c471a4a18acd21fa62cf503262302a69/natural_language_processing/extractive_question_answering/bert_base/run.py